examples/strong/analyzer-2-results.txt

```diff
@@ -1,18 +1,18 @@
 Analyzing lib, test...
-  error • 'HoneyBadger.parent' ('Root Function()') isn't a valid override of 'Animal.parent' ('Animal Function()') at lib/animal_bad.dart:15:3 • invalid_override
-  error • 'Cat.chase' ('void Function(Mouse)') isn't a valid override of 'Animal.chase' ('void Function(Animal)') at lib/animal_bad.dart:26:3 • invalid_override
+  error • 'HoneyBadger.parent' ('Root Function()') isn't a valid override of 'Animal.parent' ('parent') at lib/animal_bad.dart:15:3 • invalid_override
+  error • 'Cat.chase' ('void Function(Mouse)') isn't a valid override of 'Animal.chase' ('chase') at lib/animal_bad.dart:26:3 • invalid_override
```

Comment:
Hmm, the analyzer used to report two `Function()` types and now instead of the second function's type it repeats the second function's name? Is that expected behavior?

Reply:
No, it's a bug: https://github.com/dart-lang/sdk/issues/38238

Reply:
Thanks for the context!